commons-jetty/src/main/scala/com/avsystem/commons/jetty/rpc/JettyRPCFramework.scala

```diff
@@ -22,40 +22,25 @@ trait JettyRPCFramework extends StandardRPCFramework {
   type Reader[T] = GenCodec[T]
   type Writer[T] = GenCodec[T]
 
-  private val ArgListCodec: GenCodec[List[List[RawValue]]] = new GenCodec[List[List[String]]] {
-    private val RawValueCodec: GenCodec[RawValue] = GenCodec.StringCodec
-    private val ArgListsField = "argLists"
+  private def argListToRaw(a: List[List[RawValue]]): RawValue = {
+    def listToString(l: List[_]): RawValue = l.mkString("[", ",", "]")
```

Comment:
If you want to be overly consistent then you can also use `JsonStringOutput` here.